### PR TITLE
Example category rehaul

### PR DESCRIPTION
**Pull request by nikolausWest**
### What
- WIP rehaul of the example categories to work with the new design where all examples are shown on one big page with groups of examples separated by category title and description. WIP landing page PR: https://github.com/rerun-io/landing/pull/757
- Added some example that weren't previously listed (e.g. blueprint_stock)
- Added a script to check that all of our examples are indeed listed in the manifest (or explicitly excluded).

* Closes https://github.com/rerun-io/rerun/issues/5665

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5599/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5599/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5599/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5599)
- [Docs preview](https://rerun.io/preview/8635155f3afd9eeb2146c0e8f2e05bd4a0eb233e/docs) 
- [Examples preview](https://rerun.io/preview/8635155f3afd9eeb2146c0e8f2e05bd4a0eb233e/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: examples/manifest.toml [hunk 1]
```diff
@@ -6,105 +6,126 @@
...
+
+# These are examples that we explicitely exclude from our website. You can check that all examples are either included
+# or explicitely excluded by running `python scripts/check_example_manifest_coverage.py`.
+[ignored]
+examples = [
+  "custom_collection_adapter",
+  "custom_data_loader",
+  "custom_space_view",
+  "custom_store_subscriber",
+  "dna",
```
Comment (nikolausWest) [inline examples/manifest.toml:121]: Is actually used
```suggestion

```

File: examples/manifest.toml [hunk 2]
```diff
@@ -6,105 +6,126 @@
...
-  "dicom_mri",
-  "face_tracking",
-  "gesture_detection",
-  "human_pose_tracking",
-  "lidar",
-  "live_camera_edge_detection",
-  "live_depth_sensor",
-  "llm_embedding_ner",
-  "nuscenes",
+  "arflow",
```
Comment (nikolausWest) [inline examples/manifest.toml:17]: Move to last position (ugly) screenshot
```suggestion
 
```

File: examples/manifest.toml [hunk 3]
```diff
@@ -6,105 +6,126 @@
...
-  "rgbd",
-  "ros_node",
-  "rrt-star",
-  "segment_anything_model",
-  "signed_distance_fields",
-  "structure_from_motion",
+  "simplerecon",
+  "slahmr",
   "vrs",
+  "widebaseline",
```
Comment (nikolausWest) [inline examples/manifest.toml:23]: arflow -> last position
```suggestion
  "widebaseline",
  "arflow",
```

File: examples/manifest.toml [hunk 4]
```diff
@@ -6,105 +6,126 @@
...
 order = 3
-title = "Paper Visualizations"
+title = "Generative computer vision and LLMs"
 prelude = """
-The following examples use Rerun to create visual walkthroughs of papers. They are typically forks
-from the official open-source implementations adding Rerun as the visualizer.
-Check out the respective READMEs for installation instructions.
-For the simplest possible examples showing how to use each api,
-check out [Types](/docs/reference/types).
+Examples related to generative AI.
```
Comment (nikolausWest) [inline examples/manifest.toml:47]:
```suggestion
Examples using generative AI methods like diffusion and LLMs.
```

File: examples/manifest.toml [hunk 5]
```diff
@@ -6,105 +6,126 @@
...
-To reproduce the paper visualizations check out the README of the respective
-Rerun forks.
+[categories.integrations]
+order = 6
+title = "Integrations"
+prelude = """
+Integration with 3rd party tools, formats, libraries, and APIs.
 """
-examples = []
+examples = ["ros_node", "vrs", "eigen_opencv", "dicom_mri", "blueprint_stocks"]
```
Comment (nikolausWest) [inline examples/manifest.toml:94]: Add arflow
```suggestion
examples = [
    "ros_node",
    "vrs",
    "eigen_opencv",
    "dicom_mri",
    "blueprint_stocks",
    "arflow",
]
```

Comment (nikolausWest) [reply examples/manifest.toml:94]: Related, don't know if blueprint_stocks really fits here but maybe it's the best for now 🤷

**Review by nikolausWest — COMMENTED** [review]
Awesome! (I'd approve if github let me). Some suggestions for improvements but should be ready to just use.

For this or a later PR it would be great to add a last category called "API usage examples", which doesn't contain any examples but links to https://www.rerun.io/docs/reference/types/archetypes.

Another follow up PR should probably go through and clean up the formatting of all the tags. @martenbjork thinks it looks much better to for with "Pinhole Camera" rather than "pinhole-camera" so we should probably just take a pass and fix that.

File: examples/manifest.toml [hunk 6]
```diff
@@ -27,84 +27,85 @@ examples = [
...
 """
-examples = ["nuscenes", "ros_node", "rrt-star"]
+examples = [
+  "ros_node",
+  "nuscenes",
+  "live_depth_sensor",
+  "lidar",
+  "rrt-star",
+  "rgbd",
+  "nv12",
```
Comment (emilk) [inline examples/manifest.toml]: Unless we have a strong reason not to, I suggest we sort these lists lexicographically

Comment (abey79) [reply examples/manifest.toml]: That is order in which the examples are displayed on the web page, so the order should be deliberate, not lexicographic. I did try to make a pass on that order, putting the nicer/more relevant/etc. one in front.

Comment (emilk) [reply examples/manifest.toml]: 👍 Then also add a comment here to then to explain that motivation to the `manifest.toml` reader

Comment (abey79) [reply examples/manifest.toml]: 👍🏻

**Comment by abey79** [comment]
> Another follow up PR should probably go through and clean up the formatting of all the tags. @martenbjork thinks it looks much better to for with "Pinhole Camera" rather than "pinhole-camera" so we should probably just take a pass and fix that.

It seems the frontend does that automatically. We should do the same  in app though.

**Comment by abey79** [comment]
> For this or a later PR it would be great to add a last category called "API usage examples", which doesn't contain any examples but links to https://www.rerun.io/docs/reference/types/archetypes.

This one seems tricky. We probably need to either codegen READMEs, or some custom treatment on `landing`.

Relatedly, I think we shouldn't first focus on linking all example that make use or primitive XXX in the reference doc of XXX. This feels more useful than the other way round?

**Pull request merged**